store/eigenda.go

```diff
@@ -13,10 +13,12 @@ import (
 )
 
 type EigenDAStoreConfig struct {
-	MaxBlobSizeBytes     uint64
+	MaxBlobSizeBytes uint64
+	// the # of Ethereum blocks to wait after the L1BlockReference # before attempting to verify
```

Comment:
is L1BlockReference an eigenda or rollup reference here?
Would this flag being > 0 lead to potentially blocking behavior of some endpoints?

Reply:
It's an eigenda reference for the L1 block at which the blob batch was bridged to

Reply:
Ah can we change the name then to be more explicit? EigendaBatchL1BLockReference or something?
Otherwise it's confusing as could be interpreted like the L1origin of L2 blocks, among other things

Reply:
the variable name is embedded into the disperser response object 😅

Reply:
but can definitely update the comment